### ISSUE TITLE
iptorrents provider seems to never find torrents

### ISSUE THREAD

**Issue by dogfish182** [RuudBurger/CouchPotatoServer#5008]
### Steps to reproduce:
1. setup iptorrents as only provider
2. mark anything wanted, it never finds anything.

### Information:
Movie(s) I have this with: 
Quality of the movie being searched: default 'best' but also tried 'screener' and all other combinations
Providers I use: ONLY iptorrents
Version of CouchPotato: git:(RuudBurger:CouchPotatoServer master) 69825f9d (2015-05-27 14:06:41)
Running on: win 8.1, python 2.7.9

### Logs:
```
05-30 15:28:03  INFO [hpotato.core.plugins.base] Opening url: get https://iptorrents.eu/t?l48=&q=%22the+lazarus+effect%22+2015&qf=#torrents&p=1, data: [] 
```
05-30 15:28:03INFO[ie.providers.info.omdbapi] Found: The Lazarus Effect (2015)
05-30 15:28:03INFO[core.media.movie.searcher] Search for The Lazarus Effect in 720p ignoring ETA
05-30 15:28:03INFO[hpotato.core.plugins.base] Opening url: get https://iptorrents.eu/t?l48=&q=%22the+lazarus+effect%22+2015&qf=#torrents&p=1, data: []
05-30 15:28:04INFO[core.media.movie.searcher] Search for The Lazarus Effect in 1080p ignoring ETA
05-30 15:28:04INFO[core.media.movie.searcher] Search for The Lazarus Effect in BR-Rip ignoring ETA
05-30 15:28:04INFO[hpotato.core.plugins.base] Opening url: get https://iptorrents.eu/t?l7=&l48=&l20=&q=%22the+lazarus+effect%22+2015&qf=#torrents&p=1, data: []
05-30 15:28:05INFO[core.media.movie.searcher] Search for The Lazarus Effect in DVD-Rip ignoring ETA
05-30 15:28:05INFO[core.media.movie.searcher] Search for The Lazarus Effect in Screener ignoring ETA

---

I should note that I specifically picked a movie that is new with all available releases.

**Comment by CouchPotatoBot** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [x] Fill in all the dots. They're there for a reason
- [x] Post logs, either inline (for smaller logs) or using [Pastebin](http://pastebin.com/)
- [x] Please wrap inline logs inside ``` for readability
- [ ] Enabled debug logging in Settings > Advanced (restart CP, be sure to disable after the bug is fixed)
- [x] Post full version information

Please make sure you also read [contribution guide](https://github.com/RuudBurger/CouchPotatoServer/blob/develop/contributing.md#issues) and followed all the steps. 
Make the title describe your issue. Having "CP not working" or "I get this bug" for 100 issues, isn't really helpful. My master will close issues if there isn't enough information. On a good day he will tag the issue on close with the reason (like `can't reproduce`), but usually he won't, the lazy asshat.

Sometimes my master seems like a grumpy cat and responds with short answers. This isn't (always) because he hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let him know. Maybe he can teach me his awesome ways.

Thanks!

**Comment by dogfish182** [comment]
closing this, found in the history that 
from your python dir 

pip install lxml

would sort it out. which it did after a couple restarts of couch potato. 

this thing rocks!

**Comment by jrochet** [comment]
This is broken in 3.0.1 on Windows.  Couchpotato, which installs into %appdata%\Roaming (not sure why there's no choice), includes its own Python, along with lxml html5lib, and any other of the items that sometimes people claim resolves this issue for them.  I don't believe 3.0.1 works with IPT and I don't think there's anything anyone can do about except wait for a new version of CP which re-re-re-resolves this.

**Comment by jrochet** [comment]
I went away from using the Windows installer version (3.0.1) and instead used the run from source method which was outlined pretty nicely by you guys.  That version info is below (current as of Sept 6th it appears), and it works like a charm.  I installed it as a service with nssm and that also works great. 


Version: 4a0999d9d2a323a4005e88bffb47e6918ae51e14 (06/09/2016, 10:53:22)
Updater: source, master
ID: 11100

Thanks very much, I'll go close my own debug report.
